src/components/Picker/Picker.android.tsx

```diff
@@ -0,0 +1,194 @@
+import * as React from 'react';
+
+import { List, ListItem, Picker, Text, View, } from '@bluebase/components';
+import { Modal, TouchableOpacity, ViewStyle } from 'react-native';
+
+import { Theme } from '@bluebase/core';
+
+export interface ItemsProps {
+	value: string, label: string
+}
+export interface PickerStyles {
+	picker: ViewStyle,
+	overlay: ViewStyle
+	actionSheetOverlay: ViewStyle,
+}
+
+
+
+export interface PickerProps {
+	items: ItemsProps[],
+	selectedValue: string,
+	styles: PickerStyles,
+	label: string,
+	mode: 'modal' | 'actionSheet';
+	onValueChange: (data: string, index: number) => void
+}
+
+export interface PickerState {
+	selectedValueIndex?: number
+	textStyle?: string
+	modalVisible?: boolean
+	selectedValue: string
+	selected?: string
+
+}
+export class PickerComponent extends React.PureComponent<PickerProps, PickerState> {
+	constructor(props: PickerProps) {
+		super(props);
+
+		this.state = {
+			modalVisible: false,
+			selected: '',
+			selectedValue: '',
+		};
+	}
+
+	static defaultStyles = (_theme: Theme) => ({
+		container: {
+			backgroundColor: '#fff',
+			borderColor: '#ddd',
+			borderTopWidth: 0.5,
```

Comment:
Why is this hard coded

Reply:
the colors are hard coded because  android does not have any actionsheet effect so just to make it applicable  i have  made in modal and poped up through bottom

Reply:
So what happens on dark theme? We cannot hardcode Colors. Or themeing will not work